charts/datarepo/ci/testing-values.yaml

```diff
@@ -12,6 +12,16 @@ create-secret-manager-secret:
         - kubeSecretKey: applicationsecret
           path: secret/dsde/datarepo/integration/helm-azure-integration
           vaultKey: applicationsecret
+    - secretName: test-synapse-user
```

Comment:
Would we not want to put these all in the test-azure key and just add vals?

Reply:
Yes! I had originally tried to do that, but that's what was failing w/ the chart test action run. But, should be fixed now!